kafka-rest/src/main/java/io/confluent/kafkarest/response/StreamingResponse.java

```diff
@@ -218,11 +225,15 @@ private void closeAll(AsyncResponseQueue responseQueue) {
     responseQueue.close();
   }
 
-  private ResultOrError handleNext(T result, @Nullable Throwable error) {
+  private ResultOrError handleNext(
+      T result, @Nullable Throwable error, ProduceRecordErrorCounter produceRecordErrorCounter) {
     if (error == null) {
+      produceRecordErrorCounter.incrementErrorCount(HTTP_SUCCESS_CODE);
```

Comment:
Do we want to spend the time logging success error codes?

Reply:
The benefit of keeping success error code is that we can know the total number of produce record too. Should I still remove this line?

Reply:
yes please - I saw you did this in the airlock PR already, thank you